CommonTools/Utils/interface/KinematicTables.h

```diff
@@ -27,6 +28,72 @@ namespace edm {
       return LazyResult(&makeEtaPhiTable<Object>, objects);
     }
 
+    using TrackTable = edm::soa::Table<col::Px,
+                                       col::Py,
+                                       col::Pz,
+                                       col::P,
+                                       col::PtError,
+                                       col::MissingInnerHits,
+                                       col::NumberOfValidHits,
+                                       col::Charge,
+                                       col::Eta,
+                                       col::Phi,
+                                       col::Pt,
+                                       col::Theta,
+                                       col::D0>;
+    using TrackTableView = edm::soa::TableView<col::Px,
```

Comment:
Hi @Dr15Jones, do you have any suggestion on how to avoid a duplicate listing of the columns here? I was looking for an interface like `using TrackTableView = TrackTable::FullView` or something, but could not find it. Same problem further down down the row. Thanks a lot in advance!

Reply:
How about
```C++
template<typename T> struct ViewFromTable ;

template<typename... T> struct ViewFromTable<edm::soa::Table<T...>> {
   using type = edm::soa::View<T...>;
};
template<typename T>
using ViewFromTable_t = ViewFromTable<T>::type;

using TrackTableView =ViewFromTable<TrackTable>_t;
```

Reply:
Looks good, thanks! For the row, I found out in the meantime that I could do
```C++
using TrackRowView = TrackTable::const_iterator::value_type;
```
Is that ok? Or did you intend these lowercase typenames for internal use only?

Reply:
Sorry I forgot to ask: are you fine with me just plugging your suggestion into this PR, or you want to integrate in in another PR?

Reply:
> Or did you intend these lowercase typenames for internal use only?

The types are meant to be publicly accessible so are fine to use.

Reply:
> Sorry I forgot to ask: are you fine with me just plugging your suggestion into this PR, or you want to integrate in another PR? Would you put it into FWCore/SOA/interface/TableView.h or in the CommonTools?

`TableView.h` seems like the most logical place for `ViewFromTable`. As for what PR, that isn't really up to me anymore. @makortel what would you want?

Reply:
I'm fine with doing the changes in this PR.